### DISCUSSION TITLE
More clearer message types

### DISCUSSION THREAD

**Discussion by jhen0409** [apollographql/apollo-client-devtools#1036] [Ideas]
https://github.com/apollographql/apollo-client-devtools/blob/main/src/extension/constants.ts

The message type is used common words with no any specific prefix, I think it's possible to break some apps.

Here is what I log `window.addEventListener("message", ...)` on a webpage:
<img width="720" alt="Screenshot 2023-07-26 at 2 19 36 PM" src="https://github.com/apollographql/apollo-client-devtools/assets/3001525/e6188a5b-7468-438b-8a48-01c1a25c491f">

Could we accept a PR about add prefix like `ac-devtools:`?

Background:
I'm trying to upgrade apollo-client-devtools for [react-native-debugger](https://github.com/jhen0409/react-native-debugger) (for old remote debugger), I need to build another bridge for WebWorker to support it. However, the current message type is not easy to identify where it comes from, and it is not easy to maintain.

UPDATE: PR is here - https://github.com/apollographql/apollo-client-devtools/pull/1037

**Comment by jerelmiller** [comment]
Hey @jhen0409 👋 

This has been addressed in https://github.com/apollographql/apollo-client-devtools/pull/1226 which was released with [4.7.0](https://github.com/apollographql/apollo-client-devtools/releases/tag/v4.7.0) of the devtools. The message now contains a `source: 'apollo-client-devtools'` property for all messages sent to/from the extension.

As such I'm going to go ahead and close this discussion. Thanks for reporting this to make sure I addressed this in that PR!
